diff --git a/lib/snapshot-manager.js b/lib/snapshot-manager.js
index 68d7f8af7e1f0c74b03ddcc0c40ba4d72d57cfef..8dde8d8e5903cfd2358f169af3140fa1c32e47b4 100644
--- a/lib/snapshot-manager.js
+++ b/lib/snapshot-manager.js
@@ -97,7 +97,7 @@ function formatEntry(snapshot, index) {
 
 	const blockquote = label.split(/\n/).map(line => '> ' + line).join('\n');
 
-	return `${blockquote}\n\n${indentString(description, 4)}`;
+	return `${blockquote}\n\n\`\`\`\n${indentString(description, 0)}\n\`\`\``;
 }
 
 function combineEntries({blocks}) {
