diff --git a/lib/snapshot-manager.js b/lib/snapshot-manager.js
index 1be8a1c4d87e03416ac63531776ab960db6e831d..d61399bccd5c3b256555feff05070c7f58bcd131 100644
--- a/lib/snapshot-manager.js
+++ b/lib/snapshot-manager.js
@@ -97,7 +97,7 @@ function formatEntry(snapshot, index) {
 
 	const blockquote = label.split(/\n/).map(line => '> ' + line).join('\n');
 
-	return `${blockquote}\n\n${indentString(description, 4)}`;
+	return `${blockquote}\n\n\`\`\`\n${indentString(description, 0)}\n\`\`\``;
 }
 
 function combineEntries({blocks}) {
